index.html

```diff
@@ -122,6 +159,165 @@ <h2>Methods</h2>
+<<<<<<< HEAD
+              is not present in <extlist>, throw an IllegalArgument error.
+=======
+              is not present in <var>extlist</var>, throw an IllegalArgument error.
+>>>>>>> 441d821224ddbc98724c985e9639cffe2e20344e
+            </li>
+            <li>
+              Set the [[\HeaderExtensionsToOffer]] to <var>extlist</var>.
+            </li>
+          </ol>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>HeaderExtensionsNegotiated, readonly</dt>
+        <dd>Returns the value of the [[\HeaderExtensionsNegotiated]] slot.</dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+      <p>
+        In the <a>final steps to create an offer</a>, add a step:
+      </p>
+      <ol>
+        <li>If the [[\HeaderExtensionsToOffer]] internal slot is set, for
+          each extension in [[\HeaderExtensionsToOffer]], add an appropriate
+          "a=hdrext" line.
+        </li>
+      </ol>
+      <p>
+        In the <a>set an RTCSessionDescription</a> algorithm, add a step
+        right after the step that sets transceiver.[[\Sender]].[[\SendCodecs]],
+        saying "For each transciever, set [[\HeaderExtensionsNegotiated]] to
+        an empty list, and then
+        for each "a=hdrext" line, add an appropriate
+        RTCRtpHeaderExtensionCapability to the list.
+      </p>
+    </section>
+  </section>
+  <section id="rtcrtpheaderextensionparameters-interface">
+    <h3>
+      <dfn>RTCRtpHeaderExtensionParameters</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtpheaderextensionparameters">RTCRtpHeaderExtensionParameters</a></code>
+      dictionary is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional member.
+    </p>
+    <pre class="idl">
+      partial dictionary RTCRtpHeaderExtensionParameters {
+        boolean enabled;
+      };
+    </pre>
+    <section>
+      <h2>Attributes</h2>
+      <dl>
+        <dt>enabled, of type boolean</dt>
+        <dd>
+          <p>When returned from getParameters(), true when the RTP sender is
+            configured to send this header extension when appropriate. If the
+            attribute is missing, it means that it cannot be disabled.
+          </p>
+          <p>
+            When passed to setParameters(), true when the RTP sender should
+            send this header extension when appropriate; false when the RTP
+            sender should never send this extension.
+          </p>
+        </dd>
+      </dl>
+    </section>
+    <p>
+      The inclusion of a settable member of RTCRtpHeaderExtensionParameters
+      means that it is no longer a read-only member of RTCRtpParameters.
+      However, the "uri", "id" and "encrypted" members of
+      RTCRtpHeaderExtensionParameters are now read-only.
```

Comment:
I'd need more time to grok the overall reasons for this. Is there a short explainer I missed somewhere for what the use case is exactly? That would help me review this further.

Reply:
This particular clause just notes that the headerExtensions member of RTCRtpParameters used to be read-only, and it is no longer read-only. The sentence needs to be changed to "remain read-only", since the base spec already stated that they were read-only.